prometheus-to-sd/config/source_config.go

```diff
@@ -102,3 +123,22 @@ func SourceConfigsFromFlags(source flags.Uris, component *string, host *string,
 	}
 	return sourceConfigs
 }
+
+// LoadHTTPConfigFile parses the given YAML file into a HTTPClientConfig.
+func LoadHTTPConfigFile(filename string) (*promconfig.HTTPClientConfig, error) {
```

Comment:
Should I make these non-exported?

Reply:
Considering that they are not used outside of this package, then yes.

Reply:
Ok done.